akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/TimerScheduler.scala

```diff
@@ -15,7 +15,7 @@ import java.time.Duration
  * `TimerScheduler` is not thread-safe, i.e. it must only be used within
  * the actor that owns it.
  */
-trait TimerScheduler[T] {
+trait KeyTypedTimerScheduler[-K, T] {
```

Comment:
Not sure the rename is worth it being a breaking change for a public type even if it is apimaychange

Reply:
The class `TimerScheduler` is preserved, as a subclass of this one.  So no binary breakage, verified for Scala 2.12 with MiMa: https://github.com/lightbend/migration-manager/pull/262.